File: Arrays/Task8.java
import java.util.ArrayList;
public class Task4 {
	public static int sumofnumbersinalist(ArrayList<Integer> number) {

	int total = 0;
	for(int sum =0; sum < number.size(); sum++){
	total += number.get(sum);
	}
	return total;
}

	public static void main(String[] args){
	ArrayList<Integer> number = new ArrayList<>();

	number.add(10);
	number.add(100);
	number.add(13);
	number.add(17);
	number.add(19);
	int addition= sumofnumbersinalist(number);
	System.out.print(addition);
}
}

File: Arrays/task8.py
def total_of_a_list(number):
    """ using for loop"""
    total = 0
    for count in number:
        total += count
    return total
print(total_of_a_list([1, 2, 3, 4, 5]))

def total_of_a_list(number):
    """ using while """
    total = 0
    count = 0
    while count < len(number):
        total += number[count]
        count += 1
    return total
print(total_of_a_list([1, 2, 3, 4, 5]))


